tests/run.sh

```diff
@@ -150,34 +150,6 @@ wait_until_success() {
     if (( verbose )); then set -x; fi
 }
 
-# Wait for DISPLAY to be available, and setup xrdb,
```

Comment:
What happened to the `wait for DISPLAY to be available`-part of this? There is a non-polling magic approach to do this instead, but I'm not sure how well that works from a shell script.

Reply:
I hoped it was only necessary for `xrdb` maybe.
We could trap `SIGUSR1` in the shell script - assuming this is the parent process then.
Otherwise looping while awesome's error is `2017-11-22 15:51:47 E: awesome: main:655: cannot open display (error 1)` might also work?!

Reply:
Changed to use `xrdb -q` now.